frontend/src/components/VoteResultChart.css

```diff
@@ -1,5 +1,6 @@
 .vote-result-chart__bars {
   display: flex;
+  gap: 1px;
```

Comment:
The current 1px gap is achieved by setting `margin-left` a bit further down. I’d suggest using either `gap: 2px` or `margin-left: 2px` but not both at the same time!

Reply:
Indeed. Do you remember why we did it with margin-left previously? I now opted for gap, as I find it easier to read?

Reply:
I also checked caniuse and support is good (no opera mini and old IE versions).

Reply:
Makes sense. `gap` support for flexbox elements has been worse a few years ago, so probably that was the reason.